ui/hooks/useCurrencyRatePolling.ts

```diff
@@ -11,20 +17,66 @@ import {
 } from '../ducks/metamask/metamask';
 import usePolling from './usePolling';
 
-const useCurrencyRatePolling = () => {
-  const useCurrencyRateCheck = useSelector(getUseCurrencyRateCheck);
+const usePollingEnabled = () => {
   const completedOnboarding = useSelector(getCompletedOnboarding);
   const isUnlocked = useSelector(getIsUnlocked);
+  const useCurrencyRateCheck = useSelector(getUseCurrencyRateCheck);
+
+  return completedOnboarding && isUnlocked && useCurrencyRateCheck;
+};
+
+const useNativeCurrencies = (isPollingEnabled: boolean) => {
   const networkConfigurations = useSelector(getNetworkConfigurationsByChainId);
+  const useSafeChainsListValidation = useSelector(
+    useSafeChainsListValidationSelector,
+  );
+  const [nativeCurrencies, setNativeCurrencies] = useState<string[]>([]);
+  const chainIds = useSelector(getChainIdsToPoll);
 
-  const enabled = completedOnboarding && isUnlocked && useCurrencyRateCheck;
+  useEffect(() => {
+    // Use validated currency tickers
+    const fetchNativeCurrencies = async () => {
+      const nativeCurrenciesArray = await Promise.all(
+        Object.values(networkConfigurations).map(async (n) => {
+          const originalToken = await getOriginalNativeTokenSymbol({
+            chainId: n.chainId,
+            useAPICall: useSafeChainsListValidation && isPollingEnabled,
+          });
+
+          if (!chainIds.includes(n.chainId)) {
+            return null;
+          }
```

Comment:
This check filters our networks that are not active, or not in portfolio view.

We can remove this check if we want to add back polling for all networks.

Reply:
I assume before this was only sending  chainIds for the testnets right? (monad, sepolia and megaEth)
I think this filter is okay since for testnets we do not support the "PV feature"